Wikipedia/UI-V5/WMFHomeViewController.m

```diff
@@ -522,7 +522,10 @@ - (BOOL)tableView:(UITableView*)tableView shouldHighlightRowAtIndexPath:(NSIndex
 
 - (void)tableView:(UITableView*)tableView didSelectRowAtIndexPath:(NSIndexPath*)indexPath {
     id<WMFHomeSectionController> controller = [self sectionControllerForSectionAtIndex:indexPath.section];
-    MWKTitle* title                         = [controller titleForItemAtIndex:indexPath.row];
+    if ([controller respondsToSelector:@selector(shouldSelectItemAtIndex:)] && ![controller shouldSelectItemAtIndex:indexPath.item]) {
+        return;
```

Comment:
it doesn't matter since the highlight doesn't show. and we don't have multiple selection so the next selection will always overwrite it. And since it is a table view controller and `clearsSelectionOnViewWillAppear == YES` selection is always purged then as well.